server/routes/auth.js

```diff
@@ -38,6 +38,7 @@ router.get('/github/callback', (req, res, next) => {
     .then(response => response.data)
     .then(async githubUser => {
       if (req.session.user) {
+        await User.destroy({ where: { githubId: githubUser.id } });
```

Comment:
because if a user first login with its github account, and then it login with the regular account and wants to link to the github(which already created an user at first-time login), we don't want two user in the database holding the same githubuserId, so we destroy the first one it created and link the github info to the current one.

Reply:
Got it.  I knew there was a logical reason for it, I just wanted to understand it better.  Thanks.